app/code/Magento/SalesInventory/Model/Order/ReturnProcessor.php

```diff
@@ -17,50 +23,26 @@
 class ReturnProcessor
 {
     /**
-     * @var \Magento\CatalogInventory\Api\StockManagementInterface
-     */
-    private $stockManagement;
-
-    /**
-     * @var \Magento\CatalogInventory\Model\Indexer\Stock\Processor
+     * @var StockProcessor
      */
     private $stockIndexerProcessor;
 
-    /**
-     * @var \Magento\Catalog\Model\Indexer\Product\Price\Processor
-     */
-    private $priceIndexer;
-
-    /**
-     * @var \Magento\Store\Model\StoreManagerInterface
-     */
-    private $storeManager;
-
-    /**
-     * @var \Magento\Sales\Api\OrderItemRepositoryInterface
-     */
-    private $orderItemRepository;
-
     /**
      * ReturnProcessor constructor.
-     * @param \Magento\CatalogInventory\Api\StockManagementInterface $stockManagement
-     * @param \Magento\CatalogInventory\Model\Indexer\Stock\Processor $stockIndexer
-     * @param \Magento\Catalog\Model\Indexer\Product\Price\Processor $priceIndexer
-     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
-     * @param \Magento\Sales\Api\OrderItemRepositoryInterface $orderItemRepository
+     * @param StockManagementInterface $stockManagement
+     * @param StockProcessor $stockIndexer
+     * @param PriceProcessor $priceIndexer
+     * @param StoreManagerInterface $storeManager
+     * @param OrderItemRepositoryInterface $orderItemRepository
      */
     public function __construct(
-        \Magento\CatalogInventory\Api\StockManagementInterface $stockManagement,
-        \Magento\CatalogInventory\Model\Indexer\Stock\Processor $stockIndexer,
-        \Magento\Catalog\Model\Indexer\Product\Price\Processor $priceIndexer,
-        \Magento\Store\Model\StoreManagerInterface $storeManager,
-        \Magento\Sales\Api\OrderItemRepositoryInterface $orderItemRepository
+        private readonly StockManagementInterface $stockManagement,
+        StockProcessor $stockIndexer,
+        private readonly PriceProcessor $priceIndexer,
+        private readonly StoreManagerInterface $storeManager,
+        private readonly OrderItemRepositoryInterface $orderItemRepository
```

Comment:
A trailing comma makes future additions easier to review.
```suggestion
        private readonly OrderItemRepositoryInterface $orderItemRepository,
```